// angular2-google-maps/core/directives/google-map.ngfactory.ts
/**
 * @fileoverview This file is generated by the Angular template compiler.
 * Do not edit.
 * @suppress {suspiciousCode,uselessCode,missingProperties}
 */
 /* tslint:disable */


import * as import0 from '@angular/core';
import * as import1 from 'angular2-google-maps/core/services/google-maps-api-wrapper';
import * as import2 from 'angular2-google-maps/core/services/maps-api-loader/maps-api-loader';
import * as import3 from 'angular2-google-maps/core/directives/google-map';
import * as import4 from 'angular2-google-maps/core/services/managers/marker-manager';
import * as import5 from 'angular2-google-maps/core/services/managers/info-window-manager';
import * as import6 from 'angular2-google-maps/core/services/managers/circle-manager';
import * as import7 from 'angular2-google-maps/core/services/managers/polyline-manager';
import * as import8 from 'angular2-google-maps/core/services/managers/polygon-manager';
import * as import9 from 'angular2-google-maps/core/services/managers/kml-layer-manager';
const styles_SebmGoogleMap:any[] = ['.sebm-google-map-container-inner[_ngcontent-%COMP%] {\n      width: inherit;\n      height: inherit;\n    }\n    .sebm-google-map-content[_ngcontent-%COMP%] {\n      display:none;\n    }'];
export const RenderType_SebmGoogleMap:import0.RendererType2 = import0.ɵcrt({
  encapsulation: 0,
  styles: styles_SebmGoogleMap,
  data: {}
}
);
export function View_SebmGoogleMap_0(l:any):import0.ɵViewDefinition {
  return import0.ɵvid(0,[
    (l()(),import0.ɵted((null as any),['\n    '])),
      (l()(),import0.ɵeld(0,(null as any),(null as any),0,'div',[[
        'class',
        'sebm-google-map-container-inner'
      ]
    ],(null as any),(null as any),(null as any),(null as any),(null as any))),
    (l()(),import0.ɵted((null as any),['\n    '])),
      (l()(),import0.ɵeld(0,(null as any),(null as any),3,'div',[[
        'class',
        'sebm-google-map-content'
      ]
    ],(null as any),(null as any),(null as any),(null as any),(null as any))),
    (l()(),import0.ɵted((null as any),['\n      '])),
    import0.ɵncd((null as any),0),
    (l()(),import0.ɵted((null as any),['\n    '])),
    (l()(),import0.ɵted((null as any),['\n  ']))
  ]
  ,(null as any),(null as any));
}
function View_SebmGoogleMap_Host_0(l:any):import0.ɵViewDefinition {
  return import0.ɵvid(0,[
      (l()(),import0.ɵeld(0,(null as any),(null as any),8,'sebm-google-map',([] as any[]),[[
        2,
        'sebm-google-map-container',
        (null as any)
      ]
    ],(null as any),(null as any),View_SebmGoogleMap_0,RenderType_SebmGoogleMap)),
    import0.ɵprd(256,(null as any),import1.GoogleMapsAPIWrapper,import1.GoogleMapsAPIWrapper,[
      import2.MapsAPILoader,
      import0.NgZone
    ]
    ),
    import0.ɵdid(385024,(null as any),0,import3.SebmGoogleMap,[
      import0.ElementRef,
      import1.GoogleMapsAPIWrapper
    ]
    ,(null as any),(null as any)),
    import0.ɵprd(2304,(null as any),import4.MarkerManager,import4.MarkerManager,[
      import1.GoogleMapsAPIWrapper,
      import0.NgZone
    ]
    ),
    import0.ɵprd(2304,(null as any),import5.InfoWindowManager,import5.InfoWindowManager,[
      import1.GoogleMapsAPIWrapper,
      import0.NgZone,
      import4.MarkerManager
    ]
    ),
    import0.ɵprd(2304,(null as any),import6.CircleManager,import6.CircleManager,[
      import1.GoogleMapsAPIWrapper,
      import0.NgZone
    ]
    ),
    import0.ɵprd(2304,(null as any),import7.PolylineManager,import7.PolylineManager,[
      import1.GoogleMapsAPIWrapper,
      import0.NgZone
    ]
    ),
    import0.ɵprd(2304,(null as any),import8.PolygonManager,import8.PolygonManager,[
      import1.GoogleMapsAPIWrapper,
      import0.NgZone
    ]
    ),
    import0.ɵprd(2304,(null as any),import9.KmlLayerManager,import9.KmlLayerManager,[
      import1.GoogleMapsAPIWrapper,
      import0.NgZone
    ]
    )
  ]
  ,(ck,v) => {
    ck(v,2,0);
  },(ck,v) => {
    const currVal_0:any = true;
    ck(v,0,0,currVal_0);
  });
}
export const SebmGoogleMapNgFactory:import0.ComponentFactory<import3.SebmGoogleMap> = import0.ɵccf('sebm-google-map',import3.SebmGoogleMap,View_SebmGoogleMap_Host_0,{
  longitude: 'longitude',
  latitude: 'latitude',
  zoom: 'zoom',
  minZoom: 'minZoom',
  maxZoom: 'maxZoom',
  draggable: 'mapDraggable',
  disableDoubleClickZoom: 'disableDoubleClickZoom',
  disableDefaultUI: 'disableDefaultUI',
  scrollwheel: 'scrollwheel',
  backgroundColor: 'backgroundColor',
  draggableCursor: 'draggableCursor',
  draggingCursor: 'draggingCursor',
  keyboardShortcuts: 'keyboardShortcuts',
  zoomControl: 'zoomControl',
  styles: 'styles',
  usePanning: 'usePanning',
  streetViewControl: 'streetViewControl',
  fitBounds: 'fitBounds',
  scaleControl: 'scaleControl',
  mapTypeControl: 'mapTypeControl'
}
,{
  mapClick: 'mapClick',
  mapRightClick: 'mapRightClick',
  mapDblClick: 'mapDblClick',
  centerChange: 'centerChange',
  idle: 'idle',
  boundsChange: 'boundsChange',
  zoomChange: 'zoomChange'
}
,['*']);
