// angular2-google-maps/core/directives/google-map-info-window.ngfactory.ts
/**
 * @fileoverview This file is generated by the Angular template compiler.
 * Do not edit.
 * @suppress {suspiciousCode,uselessCode,missingProperties}
 */
 /* tslint:disable */


import * as import0 from '@angular/core';
import * as import1 from 'angular2-google-maps/core/directives/google-map-info-window';
import * as import2 from 'angular2-google-maps/core/services/managers/info-window-manager';
const styles_SebmGoogleMapInfoWindow:any[] = ([] as any[]);
export const RenderType_SebmGoogleMapInfoWindow:import0.RendererType2 = import0.ɵcrt({
  encapsulation: 2,
  styles: styles_SebmGoogleMapInfoWindow,
  data: {}
}
);
export function View_SebmGoogleMapInfoWindow_0(l:any):import0.ɵViewDefinition {
  return import0.ɵvid(0,[
      (l()(),import0.ɵeld(0,(null as any),(null as any),3,'div',[[
        'class',
        'sebm-google-map-info-window-content'
      ]
    ],(null as any),(null as any),(null as any),(null as any),(null as any))),
    (l()(),import0.ɵted((null as any),['\n      '])),
    import0.ɵncd((null as any),0),
    (l()(),import0.ɵted((null as any),['\n    '])),
    (l()(),import0.ɵted((null as any),['\n  ']))
  ]
  ,(null as any),(null as any));
}
function View_SebmGoogleMapInfoWindow_Host_0(l:any):import0.ɵViewDefinition {
  return import0.ɵvid(0,[
    (l()(),import0.ɵeld(0,(null as any),(null as any),1,'sebm-google-map-info-window',([] as any[]),(null as any),(null as any),(null as any),View_SebmGoogleMapInfoWindow_0,RenderType_SebmGoogleMapInfoWindow)),
    import0.ɵdid(385024,(null as any),0,import1.SebmGoogleMapInfoWindow,[
      import2.InfoWindowManager,
      import0.ElementRef
    ]
    ,(null as any),(null as any))
  ]
  ,(ck,v) => {
    ck(v,1,0);
  },(null as any));
}
export const SebmGoogleMapInfoWindowNgFactory:import0.ComponentFactory<import1.SebmGoogleMapInfoWindow> = import0.ɵccf('sebm-google-map-info-window',import1.SebmGoogleMapInfoWindow,View_SebmGoogleMapInfoWindow_Host_0,{
  latitude: 'latitude',
  longitude: 'longitude',
  disableAutoPan: 'disableAutoPan',
  isOpen: 'isOpen',
  zIndex: 'zIndex',
  maxWidth: 'maxWidth'
}
,{infoWindowClose: 'infoWindowClose'},['*']);
