// @ng-bootstrap/ng-bootstrap/modal/modal-backdrop.ngfactory.js
/**
 * @fileoverview This file is generated by the Angular template compiler.
 * Do not edit.
 * @suppress {suspiciousCode,uselessCode,missingProperties}
 */
/* tslint:disable */
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var import0 = require("@angular/core");
var import1 = require("@ng-bootstrap/ng-bootstrap/modal/modal-backdrop");
var styles_NgbModalBackdrop = [];
exports.RenderType_NgbModalBackdrop = import0.ɵcrt({
    encapsulation: 2,
    styles: styles_NgbModalBackdrop,
    data: {}
});
function View_NgbModalBackdrop_0(l) {
    return import0.ɵvid(0, [], null, null);
}
exports.View_NgbModalBackdrop_0 = View_NgbModalBackdrop_0;
function View_NgbModalBackdrop_Host_0(l) {
    return import0.ɵvid(0, [
        (l()(), import0.ɵeld(0, null, null, 1, 'ngb-modal-backdrop', [[
                'class',
                'modal-backdrop fade show'
            ]
        ], null, null, null, View_NgbModalBackdrop_0, exports.RenderType_NgbModalBackdrop)),
        import0.ɵdid(24576, null, 0, import1.NgbModalBackdrop, [], null, null)
    ], null, null);
}
exports.NgbModalBackdropNgFactory = import0.ɵccf('ngb-modal-backdrop', import1.NgbModalBackdrop, View_NgbModalBackdrop_Host_0, {}, {}, []);
